pkgs/by-name/ll/llama-cpp/package.nix

```diff
@@ -17,30 +16,19 @@
 , openclSupport ? false
 , clblast
 
-, openblasSupport ? !rocmSupport
+, openblasSupport ? !rocmSupport && !cudaSupport
```

Comment:
Doesn't have to be openblas (although some extra work is required to make the `blas` shim work). I think it's better to make it `blasSupport`